<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntCenterNewWidget</class>
 <widget class="QWidget" name="EntCenterNewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>696</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1016</width>
    <height>620</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1016</width>
    <height>6200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EntCenterWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#EntCenterWidget{background-color: rgb(255, 255, 255);};</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="EntLeftWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>680</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>680</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#EntLeftWidget{background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #EDF4FF, stop:1 #FAFCFF);border-top-left-radius:8px;border-bottom-left-radius:8px;border-right:1px solid #EBEBEB;}</string>
       </property>
       <widget class="QToolButton" name="switchButton">
        <property name="geometry">
         <rect>
          <x>75</x>
          <y>229</y>
          <width>100</width>
          <height>38</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>38</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>38</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton{background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(2, 164, 253, 1), stop:1 rgba(31, 139, 237, 1));border-radius:4px;font: 14px &quot;微软雅黑&quot;;color:rgba(255, 255, 255, 1);padding-left:8px;}
QToolButton:hover,pressed{ background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(40, 178, 253, 1),stop:1 rgba(64, 156, 240, 1));
		  border-radius:4px;font: 14px &quot;微软雅黑&quot;;color:rgba(255, 255, 255, 1);}</string>
        </property>
        <property name="text">
         <string>切换企业</string>
        </property>
        <property name="icon">
         <iconset resource="EntCenterWidget.qrc">
          <normaloff>:/EntCenterWidget/image/swtich.png</normaloff>:/EntCenterWidget/image/swtich.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>14</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
       <widget class="AvatarLabel" name="labelAvatar">
        <property name="geometry">
         <rect>
          <x>95</x>
          <y>59</y>
          <width>60</width>
          <height>60</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/EntCenterWidget/image/img_tx_d.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="copyTaxButton">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>162</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{border-image: url(:/EntCenterWidget/image/copy.png);}
QPushButton:hover,pressed{border-image: url(:/EntCenterWidget/image/copy_sel.png);}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="labelTaxNo">
        <property name="geometry">
         <rect>
          <x>44</x>
          <y>163</y>
          <width>150</width>
          <height>12</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>12</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>12</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12px &quot;微软雅黑&quot;;
color:#333333;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="labelSpaceLeft">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>287</y>
          <width>250</width>
          <height>1</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>1</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>1</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(222, 222, 222);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="labelMore">
        <property name="geometry">
         <rect>
          <x>29</x>
          <y>332</y>
          <width>57</width>
          <height>15</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>57</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>57</width>
          <height>15</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font:14px &quot;微软雅黑&quot;;
color:#333333;</string>
        </property>
        <property name="text">
         <string>更多</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelEntInfo">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>371</y>
          <width>10</width>
          <height>10</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border:2px solid rgba(223,70,100,1);
border-radius:5px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="labelMember">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>414</y>
          <width>10</width>
          <height>10</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border:2px solid rgba(90,168,255,1);
border-radius:5px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="mainButton">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>360</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>57</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>570</width>
          <height>30</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{background-color:rgba(255,255,255,0);font: 14px &quot;微软雅黑&quot;;color:rgba(51, 51, 51, 1);padding-left:-20px;}
QPushButton:hover,pressed{ background-color:rgba(255,255,255,0);font: 14px &quot;微软雅黑&quot;;color:rgba(31,139,237,1);}</string>
        </property>
        <property name="text">
         <string>首        页</string>
        </property>
       </widget>
       <widget class="QPushButton" name="memberButton">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>403</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>57</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>570</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{background-color:rgba(255,255,255,0);font: 14px &quot;微软雅黑&quot;;color:rgba(51, 51, 51, 1);padding-left:-20px;}
QPushButton:hover,pressed{ background-color:rgba(255,255,255,0);font: 14px &quot;微软雅黑&quot;;color:rgba(31,139,237,1);}</string>
        </property>
        <property name="text">
         <string>财务成员</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelJoin2">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>195</y>
          <width>60</width>
          <height>14</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>126</width>
          <height>14</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:14px;
font-family:SourceHanSansCN-Normal,SourceHanSansCN;
font-weight:400;
color:rgba(30,139,237,1);
line-height:21px;</string>
        </property>
        <property name="text">
         <string>加入企业</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelVip">
        <property name="geometry">
         <rect>
          <x>95</x>
          <y>59</y>
          <width>60</width>
          <height>60</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/EntCenterWidget/image/qyzx_icon_vip_wjr.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="labelComName">
        <property name="geometry">
         <rect>
          <x>55</x>
          <y>135</y>
          <width>140</width>
          <height>16</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>16</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:14px;
font-family:SourceHanSansCN-Normal,SourceHanSansCN;
font-weight:400;
color:rgba(51,51,51,1);
line-height:21px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="labeluser">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>195</y>
          <width>90</width>
          <height>14</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>126</width>
          <height>14</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:14px;
font-family:SourceHanSansCN-Normal,SourceHanSansCN;
font-weight:400;
color:rgba(102,102,102,1);
line-height:21px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="labeluserroletype">
        <property name="geometry">
         <rect>
          <x>128</x>
          <y>195</y>
          <width>100</width>
          <height>14</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>126</width>
          <height>14</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:14px;
font-family:SourceHanSansCN-Normal,SourceHanSansCN;
font-weight:400;
color:rgba(102,102,102,1);
line-height:21px;</string>
        </property>
        <property name="text">
         <string>管理员</string>
        </property>
       </widget>
       <widget class="QLabel" name="labeluserline">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>195</y>
          <width>1</width>
          <height>16</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>1</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1</width>
          <height>16</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgba(51,51,51,1);
opacity:0.3;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="labelEntInfo_2">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>457</y>
          <width>10</width>
          <height>10</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border:2px solid rgba(223,70,100,1);
border-radius:5px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="entInfoButton">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>446</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>57</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>570</width>
          <height>30</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{background-color:rgba(255,255,255,0);font: 14px &quot;微软雅黑&quot;;color:rgba(51, 51, 51, 1);padding-left:-20px;}
QPushButton:hover,pressed{ background-color:rgba(255,255,255,0);font: 14px &quot;微软雅黑&quot;;color:rgba(31,139,237,1);}</string>
        </property>
        <property name="text">
         <string>企业资料</string>
        </property>
       </widget>
       <widget class="QLabel" name="labeJoin1">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>195</y>
          <width>120</width>
          <height>14</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>126</width>
          <height>14</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:14px;
font-family:SourceHanSansCN-Normal,SourceHanSansCN;
font-weight:400;
color:rgba(102,102,102,1);
line-height:21px;</string>
        </property>
        <property name="text">
         <string>您还未加入该企业，</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="EntRightWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>750</width>
         <height>680</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>750</width>
         <height>680</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#EntRightWidget{background-color: rgb(252, 252, 252);border-top-right-radius:8px;border-bottom-right-radius:8px;}</string>
       </property>
       <layout class="QHBoxLayout" name="rightLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="layout">
          <property name="spacing">
           <number>0</number>
          </property>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>AvatarLabel</class>
   <extends>QLabel</extends>
   <header location="global">AvatarLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="EntCenterWidget.qrc"/>
 </resources>
 <connections/>
</ui>
